service.subtitles.subscene/service.py

```diff
@@ -456,9 +460,9 @@ def download(link, episode=""):
 
         if len(subtitle_list) == 0:
             if episode:
-                xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32002))).encode('utf-8'))
+                xbmc.executebuiltin((u'Notification(%s,%s)' % (SCRIPT_NAME, LANGUAGE(32002))).encode('utf-8'))
```

Comment:
can you replace by `xbmcgui.Dialog().notification()`?

Reply:
Thanks for the review. Done for all of them. 👍